Artifacts/PgSql/Structure/Security/0050-Tables-MultipleAuthStrats.sql

```diff
@@ -0,0 +1,126 @@
+(
+	ClaimSetResourceClaimActionId 
+));
+
+CREATE INDEX IF NOT EXISTS IX_ClaimSetResourceClaimActions_ActionId
+    ON dbo.ClaimSetResourceClaimActions(ActionId);
+
+CREATE INDEX IF NOT EXISTS IX_ClaimSetResourceClaimActions_ClaimSetId
+    ON dbo.ClaimSetResourceClaimActions(ClaimSetId);
+
+CREATE INDEX IF NOT EXISTS IX_ClaimSetResourceClaimActions_ResourceClaimId
+    ON dbo.ClaimSetResourceClaimActions(ResourceClaimId);
+
+
+ALTER TABLE dbo.ClaimSetResourceClaimActions ADD CONSTRAINT FK_ClaimSetResourceClaimActions_Actions FOREIGN KEY(ActionId)
+REFERENCES dbo.Actions (ActionId)
+ON DELETE CASCADE;
+
+ALTER TABLE dbo.ClaimSetResourceClaimActions ADD CONSTRAINT FK_ClaimSetResourceClaimActions_ClaimSets FOREIGN KEY(ClaimSetId)
+REFERENCES dbo.ClaimSets (ClaimSetId)
+ON DELETE CASCADE;
+
+ALTER TABLE dbo.ClaimSetResourceClaimActions  ADD  CONSTRAINT FK_ClaimSetResourceClaimActions_ResourceClaims FOREIGN KEY(ResourceClaimId)
+REFERENCES dbo.ResourceClaims (ResourceClaimId);
+
+
+
+---------------------------- dbo.ClaimSetResourceClaimActionAuthorizationStrategyOverrides --------------------------------------------------------
+
+
+CREATE TABLE dbo.ClaimSetResourceClaimActionAuthorizationStrategyOverrides(
+	ClaimSetResourceClaimActionAuthorizationStrategyOverrideId SERIAL NOT NULL,
+	ClaimSetResourceClaimActionId INT NOT NULL,
+	AuthorizationStrategyId INT NOT NULL,
+ CONSTRAINT ClaimSetResourceClaimActionAuthorizationStrategyOverrides_PK PRIMARY KEY  
+(
+	ClaimSetResourceClaimActionAuthorizationStrategyOverrideId 
+));
+
+
+CREATE INDEX IF NOT EXISTS IX_ActionAuthorizationStrategyOverrides_AuthorizationStrategyId
+    ON dbo.ClaimSetResourceClaimActionAuthorizationStrategyOverrides
+	(AuthorizationStrategyId);
+
+
+CREATE INDEX IF NOT EXISTS IX_ActionAuthorizationStrategyOverrides__ClaimActionId
+    ON dbo.ClaimSetResourceClaimActionAuthorizationStrategyOverrides
+	(ClaimSetResourceClaimActionId);
+
+ALTER TABLE dbo.ClaimSetResourceClaimActionAuthorizationStrategyOverrides  ADD  CONSTRAINT FK_ActionAuthorizationStrategyOverrides_AuthorizationStrategies FOREIGN KEY(AuthorizationStrategyId)
+REFERENCES dbo.AuthorizationStrategies (AuthorizationStrategyId)
+ON DELETE CASCADE;
+
+
+ALTER TABLE dbo.ClaimSetResourceClaimActionAuthorizationStrategyOverrides  ADD  CONSTRAINT FK_ActionAuthorizationStrategyOverrides_ActionAuthorizations FOREIGN KEY(ClaimSetResourceClaimActionId)
+REFERENCES dbo.ClaimSetResourceClaimActions (ClaimSetResourceClaimActionId);
+
+------------------------------ dbo.ResourceClaimActionAuthorizationStrategies -------------------------------------------------
+
+
+CREATE TABLE dbo.ResourceClaimActionAuthorizationStrategies(
+	ResourceClaimActionAuthorizationStrategyId SERIAL NOT NULL,
+	ResourceClaimActionId INT NOT NULL,
+	AuthorizationStrategyId INT NOT NULL,
+ CONSTRAINT ResourceClaimActionAuthorizationStrategies_PK PRIMARY KEY
+(
+	ResourceClaimActionAuthorizationStrategyId 
+));
+
```

Comment:
We're missing a unique constraint/index here, as documented in earlier comment.

Reply:
I created CREATE UNIQUE INDEX here